staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -276,6 +273,19 @@ func GetHistogramVecFromGatherer(gatherer metrics.Gatherer, metricName string, l
 	if len(metricFamily.GetMetric()) == 0 {
```

Comment:
the check in 265 is unnecessary, given this `len` check.

Reply:
Good catch! Done